lca-cli/ops/ops.go

```diff
@@ -15,13 +14,12 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/openshift/assisted-image-service/pkg/isoeditor"
 	"github.com/sirupsen/logrus"
-	etcdClient "go.etcd.io/etcd/client/v3"
```

Comment:
Can also be removed from `go.mod`

Reply:
`go mod tidy` should take care of that

Reply:
forgot :( , thanks

Reply:
need to wait for https://github.com/openshift-kni/lifecycle-agent/pull/656 first

Reply:
Merged